src/main/java/org/example/jvspringbootfirstbook/controler/BookController.java

```diff
@@ -33,4 +36,10 @@ public BookDto createBook(@RequestBody
                               CreateBookRequestDto createBookRequestDto) {
         return bookService.save(createBookRequestDto);
     }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Long id) {
+        bookService.deleteById(id);
+    }
```

Comment:
I think you are missing an update endpoint implementation:
PUT: /api/books/{id} (update a specific book);